sda/host.go

```diff
@@ -612,3 +617,63 @@ func (h *Host) Rx() uint64 {
 func (h *Host) Address() string {
 	return h.workingAddress
 }
+
+//Received gives packets received
+func Received(n map[network.ServerIdentityID]network.SecureConn) uint64 {
+	var a uint64
+	for _, value := range n {
+		a = value.Rx()
+	}
+	return a
```

Comment:
it only needs to grab a single connection, as the packets sent is part of the server. This is just a haphazard way of grabbing a single part of a map without a key.